app/views/teachers/index.html.erb

```diff
@@ -9,7 +9,7 @@
       </tr>
     </thead>
     <tbody>
-      <% @validated_teachers.each do |teacher| %>
+      <% @all_teachers.each do |teacher| %>
```

Comment:
This is beyond this pr, but perhaps we need a way to filter teachers based on their status? Could be a future story. Ask Michael

Reply:
Good point! gonna make a story for that